examples/tls-in-route.yaml

```diff
@@ -32,8 +32,7 @@ spec:
     routes:
       resource: httproutes
       routeNamespaces:
-        namespaceSelector: {}
-        onlySameNamespace: false
+        from: "AllNamespaces"
```

Comment:
Reading this feels repetitive.

Is the following an improvement?

```yaml
routeNamespaces:
  from: All
```

So, instead the constants would be 'All', 'Selector' and 'Same'.

Reply:
Good call, updated with the simplified types.

Reply:
And now I actually updated the examples too.